vendor/engines/projects/lib/projects/engine.rb

```diff
@@ -4,10 +4,12 @@ class Engine < Rails::Engine
 
     config.to_prepare do
       ::AbilityExtensionManager.extensions << "Projects::AbilityExtension"
+      ::ApplicationHelper.send :include, Projects::ApplicationHelper
```

Comment:
I actually thought this would get loaded automatically if it's in `engines/xxx/app/helpers`.